### ISSUE TITLE
UnboundExecutionError: Could not locate a bind configured on mapper Mapper|User|users, SQL expression or this Session

### ISSUE THREAD

**Issue by Mstrodl** [wildcardcorp/factored#4]
I get [an error](http://pastebin.com/raw/9XwdQ3i9) when I use Factored with nginx and type in a username to the google authenticator form. When I enter it and hit next, I am brought to a page saying

    Internal Server Error
    
    The server encountered an unexpected internal server error
    
    (generated by waitress)
I don't know if it's something wrong with [my nginx vhost](http://pastebin.com/raw/4gL93FPg) or [my Factored config](http://pastebin.com/raw/dKCUU6By). Judging by the logs, I tried adding an sqlite database named "test.db" in my Factored installation (without anything in it. I just ran sqlite3 and did `.save test`)

**Comment by vangheem** [comment]
Did you initialize the db?

```
/bin/factored_initializedb develop.ini
```

https://factored.readthedocs.org/en/latest/install.html

**Comment by Mstrodl** [comment]
Yes, I did

**Comment by Mstrodl** [comment]
I forgot to mention that it didn't do this the first time I installed it. I didn't do anything different from the first time other than copying my old `develop.ini` instead of editing it by hand.

**Comment by DarkSun144** [comment]
Same issue here. How did you solve this problem? I run the server with the default sample config. `bin/factored_initializedb develop.ini
bin/factored_adduser --username test develop.ini
bin/pserve develop.ini`

`bin/factored_listusers develop.ini` works without any issues.

**Comment by vangheem** [comment]
I can't reproduce. Might help if you send the full traceback along of the error.

**Comment by DarkSun144** [comment]
Ok, enclosed the traceback.

[traceback.txt](https://github.com/wildcardcorp/factored/files/180886/traceback.txt)

**Comment by vangheem** [comment]
What does your bin/pserve file look like? Did you follow the instructions for installation?

**Comment by DarkSun144** [comment]
Yes, i followed the instructions. Ok, i don't rename the downloaded folder but i think that's not the problem.
[pserve.txt](https://github.com/wildcardcorp/factored/files/180912/pserve.txt)

**Comment by DarkSun144** [comment]
grmbl, I just installed it on another machine. Now everything seems working very well. I'll try to reinstall it maybe something went wrong during installation.

**Comment by DarkSun144** [comment]
Ok, got it. I've forgot to run `sudo apt-get install python-dev` on Ubuntu 14.04 so somehting failed during installation.
Nevertheless, thanks for your help.
